test/resource_test.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
What purpose is this test file?

The purpose of unit test is to check whether target function's behavior is correct or not.

So, I reccomed to separate this file for target function and the corresponding test functioe.

Reply:
|- mock-practice -- create_pod.go
                               |- create_pod_test.go